README.md

```diff
@@ -33,40 +31,55 @@ Available modules
 * `status`
 * `task`
 
-Installation
----
+## Installation
 
-After building the collection artifact with `make dist`, you can install the resulting `tar.gz` file with `ansible-galaxy` in the usual ways.
+### Install from Ansible Galaxy
 
-Documentation
----
+The collection is available from Ansible Galaxy, so you can install it via
 
-You can find the inline documentation of each module with `ansible_doc pulp.squeezer.pulp_<...>`.
+    $ ansible-galaxy collection install pulp.squeezer
 
-Testing
----
+### Build locally
+
+Alternatively you can building the collection artifact with
+
+    $ make dist
+
+and install the resulting `tar.gz` file with
+
+    $ ansible-galaxy collection install pulp-squeezer-<version>.tar.gz
+
+## Documentation
+
+You can find the inline documentation of each module with `ansible-doc pulp.squeezer.<module_name>`.
+
+## Testing
 
 Testing is done by running handcrafted playbooks from `tests/playbooks` while playing back prerecorded server answers.
 Using python virtual environments is recommended.
 
-Those playbooks are meant to test one specific module, and are usually structured in three sections:
+There is usually one playbook per module that it is meant to test, but that is not a hard requirement.
+
+The playbooks are usually organized in a three-act structure:
 
- * In the first part, ran against `localhost`, fixtures can be set up.
- * The second part should contain the actual tests and is executed on the virtual host `tests` to allow recording.
- This usually involves calling the module in question several times with varying parameters and verifying it's output.
- * The third part, again for `localhost`, provides the the opportunity to undo part one.
+ * In the first act, the setup, ran against `localhost`, fixtures can be prepared.
+ * The second act, the confrontation, should contain the actual tests.
```

Comment:
Is confrontation a technical term? It sounds a small bit dramatic to me. By "should" contain does that mean that at this point the user must have done some previous step to ensure they are there? 

Does this make as much sense or is there something implied that is lost:
`The second act contains the actual tests.` 

All these questions are as much for me to learn as they are for the docs.

Reply:
"the Confrontation" see above... (but it kind of matched)
I think it was "should" because i describe new to-be-added tests. But i can switch to describe the test that _are there_.